test/TestUtilities/Test.Utility/SimpleTestSetup/SimpleTestSettingsContext.cs

```diff
@@ -137,5 +137,37 @@ public static void RemoveSetting(XDocument doc, string key)
                 item.Remove();
             }
         }
+
+        // Add NetStandard.Library and NetCorePlatforms to the feed and save the file.
+        public void AddNetStandardFeeds()
+        {
+            string[] feedlist = GetNetstandrdAndNetcorePackages();
+            var section = GetOrAddSection(XML, "packageSources");
+            AddEntry(section, "NetStandardLibrary", feedlist[0]);
+            AddEntry(section, "NetCorePlatforms", feedlist[1]);
+
+            Save();
+        }
+
+        private static string[] GetNetstandrdAndNetcorePackages()
+        {
+            string[] list = new string[2];
+            DirectoryInfo currentDir = new DirectoryInfo(Directory.GetCurrentDirectory());
+            while (currentDir != null)
+            {
+                if (currentDir.GetFiles().Any(e => e.Name.Equals("NuGet.sln", StringComparison.OrdinalIgnoreCase)))
+                {
+                    // We have found the repo root.
+                    break;
+                }
+
+                currentDir = currentDir.Parent;
+            }
+
+            DirectoryInfo repoRoot = currentDir;
+            list[0] = Path.Combine(repoRoot.FullName, "packages", "netstandard.library");
```

Comment:
don't you need the version as part of the path?

Reply:
Thanks! Both the path without version and path with version works. I'm not sure if the version is going to change in future. So I use the path without version.